service/state/metrics.go

```diff
@@ -0,0 +1,35 @@
+package state
+
+import (
+	"context"
+
+	"go.opentelemetry.io/otel/metric/global"
+	"go.opentelemetry.io/otel/metric/instrument"
+	"go.opentelemetry.io/otel/metric/unit"
+)
+
+var meter = global.MeterProvider().Meter("state")
+
+func MonitorPFDs(s *Service) {
```

Comment:
What about `WithMetrics` so that all the modules on `nodebuilder` follow the same pattern? Even though it does PFD monitoring only, for now.

The main reason for `WithMetrics` because we are not planning to granular select which piece of the stack you want to monitor, so it's either `all or nothing` or `all for a module or nothing`.

Reply:
Yes, agree

Reply:
done